collectors/python.d.plugin/postgres/postgres.chart.py

```diff
@@ -1162,7 +1162,7 @@ def zero_lock_types(databases):
 
 
 def hide_password(config):
-    return dict((k, v if k != 'password' else '*****') for k, v in config.items())
+    return dict((k, v if (k != 'password' or (k == 'password' and v == None) ) else '*****') for k, v in config.items())
```

Comment:
```suggestion
    return dict((k, v if k != 'password' or not v else '*****') for k, v in config.items())
```

Simplified: we want password to be not `******` if None or ''.